src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml

```diff
@@ -264,6 +269,11 @@ tests:
           "Step 14: TH sends RemoveNetwork Command to the DUT with NetworkID
           field set to extended PAN ID of
           PIXIT.CNET.THREAD_1ST_OPERATIONALDATASET and Breadcrumb field set to 1"
+      verification: |
+          Verify that DUT sends NetworkConfigResponse to command with the following fields:
+          NetworkingStatus is success
+
+          NetworkIndex is 'Userwifi_netidx'
```

Comment:
In response NetworkIndex has to be added:

response:
values:
- name: "NetworkingStatus"
value: 0
- name: "NetworkIndex"
value: 0

Reply:
Added